### ISSUE TITLE
How does site stats count visits ?

### ISSUE THREAD

**Issue by Ainz** [Piwigo/AStat#8]
Stats for June on one of my sites
![image](https://github.com/Piwigo/AStat/assets/3930716/653014bf-d698-4205-b01f-3aff7158e180)

That site is also monitored by Matomo
June
![image](https://github.com/Piwigo/AStat/assets/3930716/285a253e-5dd0-42aa-ad6e-63a62822f613)

I do not understand the HUGE difference.

Care to elaborate on that?

**Comment by exekutive** [comment]
does Matomo filter out webcrawler bots?

**Comment by Ainz** [comment]
Yes, it should. It is a competent Google Analytics alternative. 

Edit/
A more relevant question is if Astat does filter anything at all...?

**Comment by erAck** [comment]
_Some_ well known bots can be filtered by the [No Stats For Robots](https://piwigo.org/ext/index.php?eid=762) extension. You could add a regex to its `$bots` array list in `plugins/no_stats_for_robots/main.inc.php` if a specific bot hits you violently.
